internal/project/common/renovate.go

```diff
@@ -62,16 +62,18 @@ func (r *Renovate) CompileRenovate(o *renovate.Output) error {
 	}
 
 	o.Enable()
+
 	o.CustomManagers(xslices.Map(r.CustomManagers, func(cm CustomManager) renovate.CustomManager {
 		return renovate.CustomManager{
-			CustomType:         cm.CustomType,
-			DataSourceTemplate: cm.DataSourceTemplate,
-			DepNameTemplate:    cm.DepNameTemplate,
-			FileMatch:          cm.FileMatch,
-			MatchStrings:       cm.MatchStrings,
-			VersioningTemplate: cm.VersioningTemplate,
+			CustomType:          cm.CustomType,
+			DataSourceTemplate:  cm.DataSourceTemplate,
+			DepNameTemplate:     cm.DepNameTemplate,
+			ManagerFilePatterns: xslices.Map(cm.ManagerFilePatterns, func(s string) string { return "/" + s + "/" }),
```

Comment:
do we need to do regex escape here? assuming it's a regex?

Reply:
Yeh, makes it easier to define in `kres.yaml`. Don't want users to worry about extra renovate stuff